tests/test_sync_clk_basic/Makefile

```diff
@@ -3,10 +3,12 @@ TEST_FLAGS ?=
 
 XCC_FLAGS_HS = -O3 -g -DXUD_CORE_CLOCK=600 -save-temps -DUSB_TILE=tile[0] -DLOCAL_CLOCK_INCREMENT=10000 -DLOCAL_CLOCK_MARGIN=100 \
 							-DBUS_SPEED=2 \
+							-DXUA_USE_APP_PLL=0 \
```

Comment:
Seems to be a mismatch in define name here. Assuming this will get fixed in a later PR as discussed.

Reply:
Just found that & fixed thanks